BLAS360.cabal

```diff
@@ -52,8 +52,7 @@ test-suite blas-test
       random, tasty-quickcheck, BLAS360
   type: exitcode-stdio-1.0
 
-  frameworks: Accelerate
-  extra-libraries: cblas
+  extra-libraries: blas
```

Comment:
NOTE: The framework is no longer required.   We use the libblas.a static library regardless of build OS.